spec/models/author_spec.rb

```diff
@@ -11,6 +11,6 @@
 
 require 'rails_helper'
 
-RSpec.describe Author, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
-end
+# RSpec.describe Author, type: :model do
```

Comment:
Since this file doesn't contain any tests, could you remove it instead of commenting it out?